### PR TITLE
feat: allow user to toggle layout section headers

### PR DESCRIPTION
**Pull request by xMort**
JIRA: LX-595
risk: low

---

> [!IMPORTANT]
> Please, **don't forget to run `rush change`** for the commits that introduce **new features** 🙏

---

Refer to [documentation](https://github.com/gooddata/gooddata-ui-sdk/blob/master/dev_docs/continuous_integration.md) to see how to run checks and tests in the pull request. This is the list of the most used commands:

```
extended test - backstop
```

```
extended test - integrated
extended test - isolated
extended test - record
```

### CODE REVIEW

File: examples/playground/src/index.tsx [hunk 1]
```diff
@@ -20,6 +20,7 @@ provideCreateRoot(createRoot);
 
 const rootDOMNode = document.createElement("div");
 rootDOMNode.className = "root";
+rootDOMNode.style.height = "100%";
```
Comment (xMort) [inline examples/playground/src/index.tsx:23]: Small change that puts dashboard loader into the centre of the Playground from its location at the top of the page. This bothered me for a long time 😅

**Comment by xMort** [comment]
extended test - backstop

**Comment by xMort** [comment]
extended test - backstop

**Comment by yenkins** [comment]
Link to Storybook test result: https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/2726/artifact/libs/sdk-ui-tests/backstop/output/html-report/index.html

**Comment by xMort** [comment]
extended test - backstop

**Comment by xMort** [comment]
extended test - backstop

**Review by pbenes — APPROVED** [review]

**Review by ivan-nejezchleb — APPROVED** [review]

**Pull request merged**